### ISSUE TITLE
fix this

### ISSUE THREAD

**Issue by folliejester** [Major-Thrust/alexa-bot-api#9]
![image](https://user-images.githubusercontent.com/64592521/117965473-6c3abe00-b340-11eb-8645-9c2222cb838b.png)

**Comment by itspranavajay** [comment]
Yeah

**Comment by Ashthetik** [comment]
@folliejester @MoeZilla This project has since been superseded by [Version 3](https://github.com/KazutoKashima/alexa-bot-api-v3) If there are any remaining issues, please post them in the linked repo

**Comment by BlobTheKat** [comment]
This package is deprecated. Use either v3 or clever-bot-api (both by me :P)

**Comment by aizakkusnail** [comment]
[alexa-bot-api-v3](https://github.com/KazutoKashima/alexa-bot-api-v3) has been suspended because of Cease and Desist though. Hope this doesn't happen to `clever-bot-api` as well, even though it may be not good for the Cleverbot developers.

**Comment by Ashthetik** [comment]
Hey @aizakkusnail unfortunately, any API that does not comply with the CB API, will undergo a Cease and Desist, if caught

**Comment by BlobTheKat** [comment]
@aizakkusnail my project doesnt have a github so they'd probably never be able to reach me :P

**Comment by Ashthetik** [comment]
@BlobTheKat They'd put it into NPM

**Comment by BlobTheKat** [comment]
> @BlobTheKat They'd put it into NPM

they have better luck winning the lottery 11 times in a row than me happening to find their email in my inbox

**Comment by BlobTheKat** [comment]
this project aged like milk. maybe close this issue?

**Comment by Ashthetik** [comment]
Mainly because one person working on it got arrested, Nithish disappeared (last we heard he started scam junk), and I complied with CB's C&D since I lost interest in exploitive materials 

Only person who can close it is, Nithish and he's gone lol

**Comment by BlobTheKat** [comment]
> Mainly because one person working on it got arrested, Nithish disappeared (last we heard he started scam junk), and I complied with CB's C&D since I lost interest in exploitive materials 
> 
> Only person who can close it is, Nithish and he's gone lol

Ouch, "the slow decline of Nithish", very clickbaity yt title

**Comment by BlobTheKat** [comment]
> Mainly because one person working on it got arrested, Nithish disappeared (last we heard he started scam junk), and I complied with CB's C&D since I lost interest in exploitive materials 
> 
> Only person who can close it is, Nithish and he's gone lol
> 
Ouch, _"the slow decline of Nithish"_, very clickbaity yt title

**Comment by Ashthetik** [comment]
Yeah, pretty sure I was the only person to come out of it all with a CSec contract lol

**Comment by BlobTheKat** [comment]
> Yeah, pretty sure I was the only person to come out of it all with a CSec contract lol

nice, congrats
